Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -275,7 +275,17 @@ wxDEFINE_EVENT(DOLPHIN_EVT_RELOAD_GAMELIST, wxCommandEvent);
 
 CGameListCtrl::CGameListCtrl(wxWindow* parent, const wxWindowID id, const wxPoint& pos,
                              const wxSize& size, long style)
-    : wxListCtrl(parent, id, pos, size, style), toolTip(nullptr)
+    : wxListCtrl(parent, id, pos, size, style), toolTip(nullptr),
+      columns({// {COLUMN, {default_width (without platform padding), resizability, visibility}}
```

Comment:
Looks like an elegant solution, with the bool reference for visibility...i just hope we never change the `SConfig` instance to something else, or this will break.
I'm surprised that Lint didn't yell at this table style though.

Also: can you document why `COLUMN_PLATFORM` is the only one that adds `+1`? I know it has something to do with the picture width being 32 and adding some padding or whatever (previously `platform_icon_padding`?); but having it spelled out makes it easier to comprehend.